core/notification_system.py

```
@@ -0,0 +1,596 @@
+                    message=message,
+                    priority=priority
+                )
+                results.append(result)
+            
+            return {
+                'success': True,
+                'alert_type': alert_type.value,
+                'recipient': recipient,
+                'priority': priority.value,
+                'channels': results
+            }
```

Ruff (0.13.2)

Consider moving this statement to an `else` block

(TRY300)